hi-ml-azure/src/health_azure/himl.py

```diff
@@ -481,10 +482,17 @@ def submit_run_v2(workspace: Optional[Workspace],
     assert entry_script is not None, "No entry_script has been provided"
     snapshot_root_directory = snapshot_root_directory or Path.cwd()
     root_dir = Path(snapshot_root_directory)
-    entry_script = Path(entry_script).relative_to(root_dir).as_posix()
+
+    if "-m " not in str(entry_script):
+        entry_script = Path(entry_script).relative_to(root_dir).as_posix()
+        experiment_name = effective_experiment_name(experiment_name, entry_script)
```

Comment:
Can we get a default eperiment name from the module name if using the "-m" syntax?